projects/file-manager/src/file-manager/components/functions/node/node.component.ts

```diff
@@ -1,16 +1,16 @@
-import {Component, Input, OnInit} from '@angular/core';
-import {NodeInterface} from '../../../interfaces/node.interface';
-import {Store} from '@ngrx/store';
+import { Component, Input, OnInit } from "@angular/core";
+import { NodeInterface } from "../../../interfaces/node.interface";
+import { Store } from "@ngrx/store";
 
```

Comment:
While formatting code please use single-quotes whenever possible

Reply:
Applies to any code later in files

Reply:
Can I make a suggestion to use prettier so this project won't have any code formatting issues?
If you want I can make it as a new pull request after this one and it will also format all the files.
You can also add prettier checks to the ci/cd of the project and not allow code that is not well formatted into master branch.

Reply:
@Chiff ^^^^